joinmarket/message_channel.py

```diff
@@ -255,7 +255,7 @@ def privmsg(self, nick, cmd, message, mc=None):
             self.active_channels[nick].privmsg(nick, cmd, message)
             return
         else:
-            log.debug("Failed to send message to: " + str(nick) + \
+            log.warn("Failed to send message to: " + str(nick) + \
```

Comment:
This one could happen very often e.g. if there is a spy connecting/disconnecting. I think info or debug would be fine, but I don't mind.